guides/doc-Upgrading_and_Updating/topics/post_upgrade_template_checks.adoc

```diff
@@ -0,0 +1,37 @@
+[id="post_upgrade-updating-templates-parameters_{context}"]
+= Updating Templates, Parameters, Lookup Keys and Values
+During the upgrade process, {Project} attempts to locate macros that are deprecated for {Project} {ProjectVersion} and converts old syntax to new syntax for the default {Project} templates, parameters, and lookup keys and values.
+However, {Project} does not convert old syntax in the custom templates that you have created and in the cloned templates.
+
+The process uses simple text replacement, for example:
+----
+@host.params['parameter1'] -> host_param('parameter1')
+@host.param_true?('parameter1') -> host_param_true?('parameter1')
+@host.param_false?('parameter1') -> host_param_false?('parameter1')
+@host.info['parameters'] -> host_enc['parameters']
+----
+
+[WARNING]
+If you use cloned templates in {Project}, verify whether the cloned templates have diverged from the latest version of the original templates in {Project}.
+The syntax for the same template can differ between versions of {Project}.
+If your cloned templates contain outdated syntax, update the syntax to match the latest version of the template.
+
+To ensure that this text replacement does not break or omit any variables in your files during the upgrade, check all templates, parameters, and lookup keys and values for the old syntax and replace manually.
+
+The following error occurs because of old syntax remaining in files after the upgrade:
+
+----
+ undefined method '#params' for Host::Managed::Jail
+----
+
+.Fixing the outdated subscription_manager_registration snippet
```

Comment:
This was dropped in 52b8ae194df55b5d9897278ca78a5450d1f5e53f and should also stay out